src/docs/win_index.html

```diff
@@ -19,21 +19,21 @@
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=ISO-8859-1">
   <META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">
   <LINK REL="STYLESHEET" HREF="DocIndex.css" CHARSET="ISO-8859-1" TYPE="text/css">
-  <TITLE>Pivotal&#8482; GemFire&#174; Native Client Documentation</TITLE>
+  <TITLE>Apache Geode Native Client Documentation</TITLE>
   <style type="text/css">
 <!--
 .style2 {font-size: medium}
 -->
   </style>
 </HEAD>
 <BODY bgcolor="#ffffff">
-     <IMG SRC="PIVOTAL_GemFire_190x81.png" BORDER="0">
      <h1 align="left">
      <FONT size=6><b>
-Pivotal<FONT size=6><b><sup><font size=-0>&#8482;</font></sup></b></FONT></font> GemFire<FONT size=6><b><sup><font size=-0>&#174;</font></sup></b></FONT></font> Native Client Documentation</H1>
+Apache Geode Native Client Documentation</H1>
      </b></FONT>
 <p>
 Access documentation at <a href="http://docs-gemfire-native-develop.cfapps.io" target="_blank"><b>Pivotal GemFire Native Client Documentation</b></a>.
```

Comment:
@pivotal-jbarrett until we have the docs hosted on geode.apache.org, unless you have a better suggestion.